addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -74,6 +74,86 @@
 		</config-description>
 	</thing-type>
 
+	<thing-type id="cco">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="ipbridge"/>
+		</supported-bridge-type-refs>
+
+		<label>CCO</label>
+		<description>Contact Closure Output</description>
+
+		<channels>
+			<channel id="switchstatus" typeId="switchState"/>
+		</channels>
+
+		<config-description>
+			<parameter name="integrationId" type="integer" required="true">
+				<label>Integration ID</label>
+				<description>Address of CCO in the Lutron control system</description>
+			</parameter>
+			<parameter name="defaultPulse" type="integer">
+				<label>Pulse Length</label>
+				<description>Default pulse length in seconds (0.1 - 100)</description>
+				<default>0.5</default>
```

Comment:
Yes, it definitely should be type decimal rather than integer!  I fixed that in both places, and added a min and max.  The parameter is named defaultPulse because in the future I'd like to add a way to change the pulse time with each activation.  That isn't necessary for most applications, though, so I didn't include it yet.